### ISSUE TITLE
[NBT] Random suitcases in place of the Inflatable Kits

### ISSUE THREAD

**Issue by CaptainGecko** [Aurorastation/Aurora.3#13187]
There's a rack with with two suitcases in Xenoarcheology, on the Horizon. I don't know what their point is, their presence seems a bit random. considering that there were a couple of Inflatable Kits (with inflatable doors and walls in, naturally) in the Aurora's Atrium, on a rack, I believe that these kits, on the Horizon were somehow swapped with suitcases somehow.

![unknown](https://user-images.githubusercontent.com/94259758/153963031-89dd2498-fe32-4f24-ad98-9a9467c5ce6c.png)
In other words, replace the suitcases there by inflatable kits, please!
